system/setup.py

```diff
@@ -54,12 +54,14 @@
       install I(facter) and I(ohai) means you can avoid Ruby-dependencies on your
       remote systems. (See also M(facter) and M(ohai).)
     - The filter option filters only the first level subkey below ansible_facts.
-    - If the target host is Windows, you will not currently have the ability to use
-      C(fact_path) or C(filter) as this is provided by a simpler implementation of the module.
-      Different facts are returned for Windows hosts.
+    - If the target host is Windows you can now use C(fact_path). Make sure that this path 
+      exists on the target host. Files in this path MUST be PowerShell scripts (*.ps1) and 
+      their output must be formattable in JSON (Ansible will take care of this). Test the 
```

Comment:
Would it be worth adding that for windows hosts fact_path was added in ansible 2.1 and restoring some of the text about filter not being available on windows hosts?